docs_user/modules/con_openshift-preparation-for-block-storage-adoption.adoc

```diff
@@ -34,37 +32,31 @@ spec:
 < . . . >
 ----
 +
-For more information about node selection, see xref:about-node-selector_planning[About node selector].
+For more information about node selection, see link:{defaultOCPURL}/nodes/index#nodes-scheduler-node-selectors-about_nodes-scheduler-node-selectors[About node selectors].
 +
 [NOTE]
 ====
 If your nodes do not have enough local disk space for temporary images, you can use a remote NFS location by setting the extra volumes feature ()`extraMounts`.
```

Comment:
Does the parentheses () before `extraMounts` need to be there?

Reply:
I don't think so.